### PR TITLE
create training form

### PR DESCRIPTION
**Pull request by antimage-stack**

### CODE REVIEW

**Comment by Alekskarb** [comment]
Check the naming of fields (RU)

**Review by Alekskarb — APPROVED** [review]
in generally all good

File: src/app/trainingform/trainingform.component.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      </mat-form-field>
+
+      <mat-form-field appearance="standard" class="form-control">
+        <label class="star">{{ 'trainingform.publication' | translate}}</label>
+        <input matInput [matDatepicker]="picker2">
+        <mat-datepicker-toggle matSuffix [for]="picker2"></mat-datepicker-toggle>
+        <mat-datepicker #picker2></mat-datepicker>
+      </mat-form-field>
+
+      <button mat-button disabled>{{ 'trainingform.create-btn' | translate}}</button>
```
Comment (Darya2311) [inline src/app/trainingform/trainingform.component.html:105]: Can't click the button even if all fields are completed

**Review by Darya2311 — APPROVED** [review]
It's ok

File: src/app/trainingform/trainingform.component.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        <label class="star">{{ 'trainingform.registry-duration' | translate}}</label>
+        <mat-date-range-input [rangePicker]="picker1">
+          <input matStartDate formControlName="trainingRegistryStart" placeholder="{{ 'trainingform.start-date' | translate}}">
+          <input matEndDate formControlName="trainingRegistryEnd" placeholder="{{ 'trainingform.end-date' | translate}}">
+        </mat-date-range-input>
+        <mat-datepicker-toggle matSuffix [for]="picker1"></mat-datepicker-toggle>
+        <mat-date-range-picker #picker1></mat-date-range-picker>
+      </mat-form-field>
+
+      <mat-form-field appearance="standard" class="form-control">
```
Comment (ViktorRiabinin) [inline src/app/trainingform/trainingform.component.html:98]: Should the publication date be assigned automatically at the time the entity is created?

**Pull request merged**